crates/turborepo-lib/src/commands/logout.rs

```diff
@@ -1,38 +1,59 @@
 use tracing::error;
-use turborepo_auth::logout as auth_logout;
+use turborepo_auth::{logout as auth_logout, read_or_create_auth_file, AuthToken};
 
-use crate::{cli::Error, commands::CommandBase, config, rewrite_json::unset_path};
+use crate::{cli::Error, commands::CommandBase};
 
-pub fn logout(base: &mut CommandBase) -> Result<(), Error> {
-    if let Err(err) = remove_token(base) {
-        error!("could not logout. Something went wrong: {}", err);
-        return Err(err);
+pub async fn logout(base: &mut CommandBase) -> Result<(), Error> {
+    let client = base.api_client()?;
+    let auth_path = base.global_auth_path()?;
+    let config_path = base.global_config_path()?;
+    let mut auth_file = read_or_create_auth_file(&auth_path, &config_path, &client).await?;
+
+    if auth_file.tokens().is_empty() {
```

Comment:
This is a super tiny nit, but you could do this as a `match`:

```
match auth_file.tokens().len() {
  0 => {
    println!("No tokens to remove");
    return Ok(());
  },
  1 => ...
  _ => ...
}
```

Reply:
nits help me be more Rust brained than Go brained so in this house we love super nits